.github/workflows/build-master.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: docker version && docker info
       - uses: actions/checkout@v2
       - name: build current docs
-        run: docker build --target=deploy-source --output=./_site .
+        run: docker build --build-arg JEKYLL_ENV=production --target=deploy-source --output=./_site .
```

Comment:
Wasn't sure what environment we want "master" to be built for; perhaps this should also be "development"?

Reply:
I believe it makes sense to keep master builds as dev builds.

Reply:
Yes, master should be still "development" without polldaddy, ...

Reply:
oh, let me update that 👍